pkgs/top-level/perl-packages.nix

```
@@ -10230,6 +10236,8 @@ with self; {
       description = "Locate per-dist and per-module shared files";
       homepage = "https://metacpan.org/release/File-ShareDir";
       license = with lib.licenses; [ artistic1 gpl1Plus ];
+      # Can't load module IO, dynamic loading not available in this perl.
+      broken = stdenv.hostPlatform != stdenv.buildPlatform;
```

```suggestion
      broken = !(stdenv.buildPlatform.canExecute stdenv.hostPlatform);
```

e.g. we probably don't want to mark broken if it's x86_64 with SSE 4.2 compiling to x86_64 w/ SSE 4.1.

only commenting this once but it applies to the rest of the patch too.

---

Done. Don't really know how to test it however.